pkg/plugins/resources/postgres/plugin.go

```diff
@@ -20,5 +20,20 @@ func (p *plugin) NewResourceStore(pc core_plugins.PluginContext, config core_plu
 	if !ok {
 		return nil, errors.New("invalid type of the config. Passed config should be a PostgresStoreConfig")
 	}
+	migrated, err := isDbMigrated(*cfg)
+	if err != nil {
+		return nil, err
+	}
+	if !migrated {
+		return nil, errors.New(`database is not migrated. Run "kuma-cp migrate up" to update database to the newest schema`)
```

Comment:
If the database has already been updated to a newer version, this message will be confusing and running `kuma-cp migrate up` won't help

Reply:
How so? If you try to run older version of Kuma than your DB this error is thrown. Then when you run "kuma-cp migrate up", you will see "DB is migrated to newer version than Kuma. DB migration version %d. Kuma migration version %d. Run newer version of Kuma".

Besides, I don't expect this is really common situation. It's more sort of defence mechanism.

Reply:
@yskopets 
since you approved the PR I'm merging it, because I want to move forward. If you feel like this should be approved, please create an issue on GH/Trello. Thanks.